package.json

```diff
@@ -25,6 +28,7 @@
     "@types/jest": "^25.1.4",
     "@types/node": "^12.12.31",
     "jest": "^25.1.0",
+    "prettier": "^2.0.4",
```

Comment:
[Prettier recommends pinning an exact version.](https://prettier.io/docs/en/install.html)

Reply:
I'm fine with major version binding.  Semver should hold compat on major versions.  It's also consistent with jest and the other repos.